server/apps/main/views.py

```diff
@@ -353,4 +353,4 @@ def moderate_experience(request, uuid):
                 },
             )
     else:
-        redirect("index")
+        return redirect("index")
```

Comment:
Fixed to now properly redirect to index for non-logged in and non-moderator users (rather than leading an error 500)